### PR TITLE
update CHANGELOG-3.5.md for #17447

### PR DESCRIPTION
**Pull request by halleyshx**
update CHANGELOG-3.5.md for https://github.com/etcd-io/etcd/pull/17447

### CODE REVIEW

File: CHANGELOG/CHANGELOG-3.5.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 ### Dependencies
 - Compile binaries using [go 1.21.6](https://github.com/etcd-io/etcd/pull/17362)
 
+### etcd grpc-proxy
+
+- Add [`etcd grpc-proxy start --dial-keepalive-time`](https://github.com/etcd-io/etcd/pull/17447) flag to keepalive time for client(grpc-proxy) connections.
+- Add [`etcd grpc-proxy start --dial-keepalive-timeout`](https://github.com/etcd-io/etcd/pull/17447) flag to keepalive timeout for client(grpc-proxy) connections.
+- Add [`etcd grpc-proxy start --permit-without-stream`](https://github.com/etcd-io/etcd/pull/17447) flag to enable client(grpc-proxy) to send keepalive pings even with no active RPCs.
```
Comment (ahrtr) [inline CHANGELOG/CHANGELOG-3.5.md:20]: Also put it right above the section "### Dependencies"
```suggestion
### gRPC Proxy
- Add [three flags (see below) for grpc-proxy](https://github.com/etcd-io/etcd/pull/17447)
  - `--dial-keepalive-time`
  - `--dial-keepalive-timeout`
  - `--permit-without-stream`
```

**Comment by ahrtr** [comment]
Please rebase this PR and squash the commits.

**Comment by ahrtr** [comment]
> Please rebase this PR and squash the commits.

We expect there is only one commit. THis PR has 4 commits

**Comment by ahrtr** [comment]
Please rebase this PR firstly, the squash the 4 commits into one.

**Comment by halleyshx** [comment]
> Please rebase this PR firstly, the squash the 4 commits into one.

I did it, but I don't know why it didn't work. I would to make a new pr to make it work.

**Comment by halleyshx** [comment]
@ahrtr Pls check https://github.com/etcd-io/etcd/pull/17488

**Pull request closed without merging**